### PR TITLE
Rearrange profile fields

### PR DESCRIPTION
**Pull request by victor-paumier**
- move isLocal to situation part
- move hobbies & motivations to preference part

### CODE REVIEW

File: templates/User/edit_content.html.twig [hunk 1]
```diff
@@ -137,23 +141,15 @@
...
-            {{ form_widget(edit_form.hobbies,{'attr':{'class': 'select2multiple form-control'}}) }}
-            {{ form_errors(edit_form.hobbies) }}
-        </div>
-    </div>
-    <div class="row mt-4 form-group">
-        <div class="col-12">
-            {{ form_label(edit_form.motivation, 'user.motivation'|trans) }}
-            {{ form_widget(edit_form.motivation,{'attr':{'class': 'form-control'}}) }}
-            {{ form_errors(edit_form.motivation) }}
+            {{ form_label(edit_form.languagesWanted, 'profile.languages_wanted'|trans) }}
```
Comment (JeremieSamson) [inline templates/User/edit_content.html.twig]: why not adding label in the form type ?

File: templates/User/edit_content.html.twig [hunk 2]
```diff
@@ -163,13 +159,20 @@
     </div>
     <div class="row mt-4 form-group">
         <div class="col-12">
-            {{ form_label(edit_form.languagesWanted, 'profile.languages_wanted'|trans) }}
-            {{ form_widget(edit_form.languagesWanted,{'attr':{'class': 'select2multiple form-control'}}) }}
-            {{ form_errors(edit_form.languagesWanted) }}
+            {{ form_label(edit_form.hobbies, 'user.hobbies'|trans) }}
```
Comment (JeremieSamson) [inline templates/User/edit_content.html.twig]: same here

File: templates/User/edit_content.html.twig [hunk 3]
```diff
@@ -163,13 +159,20 @@
...
+            {{ form_widget(edit_form.hobbies,{'attr':{'class': 'select2multiple form-control'}}) }}
+            {{ form_errors(edit_form.hobbies) }}
+        </div>
+    </div>
+    <div class="row mt-4 form-group">
+        <div class="col-12">
+            {{ form_label(edit_form.motivation, 'user.motivation'|trans) }}
```
Comment (JeremieSamson) [inline templates/User/edit_content.html.twig]: same here

File: templates/User/edit_content.html.twig (hunk 1 above)
Comment (victor-paumier) [reply templates/User/edit_content.html.twig]: done

File: templates/User/edit_content.html.twig (hunk 2 above)
Comment (victor-paumier) [reply templates/User/edit_content.html.twig]: done

File: templates/User/edit_content.html.twig (hunk 3 above)
Comment (victor-paumier) [reply templates/User/edit_content.html.twig]: done

**Pull request merged**